packages/scss/src/components/_input.scss

```diff
@@ -19,21 +19,21 @@
 	width: 100%;
 
 	&::placeholder {
-		color: $greyLighter;
+		color: #CCCCCC;
```

Comment:
I'd say yes... It's a very specific color usage. And I'm not sure that it's a got idea to let the placeholder color be overrided.